src/expense/expense.controller.ts

```diff
@@ -30,6 +31,7 @@ export class ExpenseController {
     status: 201,
     description: "Expense created",
   })
+  @Roles("USER")
```

Comment:
spróbuj może coś w tym stylu do wszystkich anotacji
czy pomoże? - nie wiem, ale chyba nie powinno zaszkodzić

```suggestion
  @UseGuards(AuthGuard, RoleGuard)
  @Roles(Role.User)
```